sarif/operations/summary_op.py

```diff
@@ -59,12 +58,12 @@ def _generate_summary(input_files: SarifFileSet) -> List[str]:
     of which error codes are present.
     """
     ret = []
-    result_count_by_severity = input_files.get_result_count_by_severity()
-    for severity in sarif_file.SARIF_SEVERITIES:
-        issue_code_histogram = input_files.get_issue_code_histogram(severity)
-        result_count = result_count_by_severity.get(severity, 0)
+    report = input_files.get_report()
+    for severity in report.get_severities():
+        result_count = report.issue_count_for_severity(severity)
+        issue_type_histogram = report.get_issue_type_histogram(severity)
```

Comment:
`get_issue_type_histogram` has issue counts as the values, `get_issues_grouped_by_type` has the list of issues.  So `get_issue_type_histogram` is a convenience method to avoid the need to measure the list lengths, and it's all we need for this summary which just gives issue counts.